lib/miq_automation_engine/models/miq_ae_yaml_export.rb

```diff
@@ -259,4 +264,9 @@ def swap_domain_path(fqname)
     return fqname if @options['export_as'].blank?
     fqname.gsub(/^\/#{@domain}/, @options['export_as'])
   end
+
+  def domain_accessible?
+    @tenant.editable_domains.collect(&:name).map(&:upcase).include?(@domain.upcase) ||
+      @is_root_tenant
```

Comment:
Suggest checking `@is_root_tenant` first then doing the other logic if needed.  Also no need to wrap this line.